tests/test_filters.py

```diff
@@ -631,7 +631,15 @@ def test_glyph(datasets, sphere):
         geom=geoms, scale='arr', orient='Normals', factor=0.1, tolerance=0.1, progress_bar=True
     )
     assert sphere.glyph(geom=geoms[:1], indices=[None], progress_bar=True)
-    assert sphere_sans_arrays.glyph(geom=geoms, progress_bar=True)
+
+    with pytest.warns(Warning):  # tries to orient but no orientation vector available
```

Comment:
I don't want to go overboard, but do we want to define a custom `DataWarning`/`ActiveDataWarning` or something to be emitted in these cases? :smile: (I'm still not great at naming things.)

Reply:
I'm thinking that the warning should indicate that the input parameter is changed/ignored.  This would be a general and fairly useful warning type.  I'm blanking on a concise name too.

Reply:
I am torn in two directions.

One is something like `ActiveAttributesWarning`, since we cannot set an active attribute in the vtk sense. If this too vtkish, I like your `ActiveDataWarning`.  These are the cause of the warning.

To go another way, something like `ParameterChangeWarning` to indicate the effect of what is happening.

Reply:
I think we can resolve this later, but as-is, this is sufficient.